deno/whatsapp-with-vonage/src/main.ts

```diff
@@ -0,0 +1,82 @@
+import { crypto } from "https://deno.land/std@0.203.0/crypto/mod.ts";
+import { encodeHex } from "https://deno.land/std@0.203.0/encoding/hex.ts";
+
+import { jwtVerify } from "https://deno.land/x/jose@v4.15.2/index.ts";
+
+import { throwIfMissing, getStaticFile } from "./utils.ts";
+
+type Context = {
+    req: any;
+    res: any;
+    log: (msg: any) => void;
+    error: (msg: any) => void;
+};
+
+export default async ({ req, res, log, error }: Context) => {
+    throwIfMissing(Deno.env.toObject(), [
+        "VONAGE_API_KEY",
+        "VONAGE_ACCOUNT_SECRET",
+        "VONAGE_SIGNATURE_SECRET",
```

Comment:
Can we change this to VONAGE_API_SIGNATURE_SECRET throughout including readme?

Reply:
Yeah